client/src/components/CoverPhoto.tsx

```diff
@@ -12,6 +12,7 @@ export interface CoverPhotoProps {
   className?: string;
   image: Image | null | undefined;
   size?: string;
+  ease?: boolean;
```

Comment:
Is this to prevent the fade in effect? If so, what about calling this `animate` or `animateIn`?

Reply:
Yeah, I like `animateIn` 👍  Updated in https://github.com/apollographql/spotify-showcase/pull/87/commits/7b70a9dbda1107d0b88581a2203bdd8e993ba0c4